Doc/library/operator.rst

```diff
@@ -314,6 +314,8 @@ expect a function argument.
    method.  Dictionaries accept any hashable value.  Lists, tuples, and
    strings accept an index or a slice:
 
+      >>> itemgetter('name')({'name': 'tu', 'age': 18})
```

Comment:
I think this shouldn't put here, this code block is for explaining `Lists, tuples, and strings accept an index or a slice:`, not `Dictionaries accept any hashable value`.

If we want to add a new example for dictionaries, it should make a new code block above this one.